.travis.yml

```diff
@@ -1,10 +1,10 @@
 sudo: false
 cache: bundler
 rvm:
-  - 2.2.10
```

Comment:
This gem still support ruby 2.2

Reply:
Sure, I didn't want to add too many job entries, didn't mean to drop support! Will readd.